### ISSUE TITLE
sentinel rule Template issue

### ISSUE THREAD

**Issue by TheCat1989** [Azure/Azure-Sentinel#3364]
In the sentinel analytics we can find this rule called : **Azure DevOps - Retention to Zero** 
with the following query logic:

AzureDevOpsAuditing
| where OperationName =~ "Pipelines.PipelineRetentionSettingChanged"
| where Data.SettingName in ("PurgeArtifacts", "PurgeRuns")
| where Data.NewValue == 0
| project-reorder TimeGenerated, OperationName, ActorUPN, IpAddress, UserAgent, Data
| extend timestamp = TimeGenerated, AccountCustomEntity = ActorUPN, IPCustomEntity = IpAddress

However, in the Azure DevOps Portal, it prevents the user from setting the retention value to zero
basically there is NO WAY to set the value to zero. so what is the purpose of this rule template given that it is impossible to trigger.

**Comment by aprakash13** [comment]
The query has been modified now to address the issue. 
https://github.com/Azure/Azure-Sentinel/blob/master/Detections/AzureDevOpsAuditing/ADORetentionReduced.yaml
